### PR TITLE
Mute/Unmute by FAB revised

### PR DESCRIPTION
**Pull request by hasevr**
Based on comments on "Feature/mute by fab", I've create a revised version.
Now, the appearance of both remote and local participants  is changed by mute/unmute.

### CODE REVIEW

File: services/conference/src/scripts/components/footer/footer.tsx [hunk 1]
```diff
@@ -65,7 +67,7 @@ export const Footer: React.FC<BaseProps> = (props) => {
         <MenuItem onClick={closeMicMenu}>Mic 3</MenuItem>
       </Menu>
       <Fab className={classes.margin} size = "small" color={store.cameraOn ? 'secondary' : 'primary'}
-         aria-label="camera"onClick = { () => { store.cameraOn = !store.cameraOn }}>
+          aria-label="camera"onClick = { () => { store.cameraOn = !store.cameraOn }}>
```
Comment (zlguo1996) [inline services/conference/src/scripts/components/footer/footer.tsx:70]: This button should be disabled when user forbiden camera or have no camera.

Comment (zlguo1996) [reply services/conference/src/scripts/components/footer/footer.tsx:70]: Could directly use plugins.streamControl.muteVideo of local participant store.

File: services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+(micOn) => {
+  const tracks = connection.localTracks
+  for (const track of tracks) {
+    if (track.getType() === 'audio') {
+      micOn ? track.unmute() : track.mute()
+    }
+  }
+},
+)
+reaction(() => appLevel.cameraOn,
```
Comment (zlguo1996) [inline services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts:16]: no need to seperate `appLevel.cameraOn` and `participants.local.get().plugins.streamControl.muteVideo`
```
reaction(() => participants.local.get().plugins.streamControl.muteVideo, ...
```

File: services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  (cameraOn) => {
+    const tracks = connection.localTracks
+    for (const track of tracks) {
+      if (track.getType() === 'video') {
+        cameraOn ? track.unmute() : track.mute()
+        participants.local.get().plugins.streamControl.muteVideo = !cameraOn
```
Comment (zlguo1996) [inline services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts:22]: this line is not needed anymore

File: services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts (hunk 2 above)
Comment (zlguo1996) [reply services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts:16]: In this case, `micOn` and `cameraOn` in AppLevel is not needed anymore.

Comment (hasevr) [reply services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts:16]: I also considered about it. But when leave from one conference and join to another conference, is 'participants.local.get().plugins.streamControl.muteVideo' kept ?
If so, I'll remove appLevel store.

**Comment by hasevr** [comment]
I'm adding mic and camera selection also. So, this will come with that.

**Pull request closed without merging**